src/map/entities/mobentity.cpp

```diff
@@ -683,30 +683,183 @@ void CMobEntity::DistributeRewards()
     }
 }
 
+// Return the list of seals that can drop based on the mob's level.
+// Rules:
+// - Mob  < 50: Beastmen's Seal
+// - Mob >= 50: Beastmen's Seal, Kindred's Seal
+// - Mob >= 70: Beastmen's Seal, Kindred's Seal, Kindred's Crest
+// - Mob >= 80: Beastmen's Seal, Kindred's Seal, Kindred's Crest, High Kindred's Crest
+// If Abyssea is not enabled, pool is limited to Beastmen's Seal and Kindred's Seal.
+auto CMobEntity::GetEligibleSeals() -> std::vector<uint16>
+{
+    constexpr uint16 BEASTMENS_SEAL      = 1126;
+    constexpr uint16 KINDREDS_SEAL       = 1127;
+    constexpr uint16 KINDREDS_CREST      = 2955;
+    constexpr uint16 HIGH_KINDREDS_CREST = 2956;
```

Comment:
You can extract these constants into an anon namespace at the top of the cpp file:
```cpp
namespace
{
    constexpr uint16 BEASTMENS_SEAL      = 1126;
    constexpr uint16 KINDREDS_SEAL       = 1127;
    constexpr uint16 KINDREDS_CREST      = 2955;
    constexpr uint16 HIGH_KINDREDS_CREST = 2956;
}
```